aws-fs-lambda-webserver/pulumi/Program.fs

```diff
@@ -7,7 +7,12 @@ open Pulumi.Aws.Lambda
 
 module ManagedPolicies =
     let AWSLambdaBasicExecutionRole = "arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole"
-    let AWSLambdaFullAccess         = "arn:aws:iam::aws:policy/AWSLambdaFullAccess"
+    let AWSLambdaExecute         = "arn:aws:iam::aws:policy/AWSLambdaExecute"
+
+[<AutoOpen>]
+module Helpers =
+    let inputLeft<'a, 'b>(v: 'a) : InputUnion<'a, 'b> = InputUnion.op_Implicit v
+    let inputRight<'a, 'b>(v: 'b) : InputUnion<'a, 'b> = InputUnion.op_Implicit v
```

Comment:
We have standard helpers to do that https://github.com/pulumi/pulumi/blob/master/sdk/dotnet/Pulumi.FSharp/Library.fs#L39-L44